From 5e248c400d5df4a4425455678b7e92513bc5eb3c Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Mon, 4 May 2020 17:24:16 -0700
Subject: [PATCH] dgit-maint-merge(7): Mention --allow-unrelated-histories

This won't come up often so don't spend too many words on it.

It can be needed when converting an existing package with no existing
git history, in which case the upstream git history and the dgit
synthesised history are considered unrelated by git.

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-merge.7.pod | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index 51b6a69d..c3931ebe 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -426,6 +426,9 @@ Once you're satisfied with what will be merged, update your package:
 
 =back
 
+The first time you do this you might need to pass
+I<--allow-unrelated-histories> to git-merge(1).
+
 If you obtained a tarball from upstream, you are ready to try a build.
 If you merged a git tag from upstream, you will first need to generate
 a tarball:
-- 
2.26.2

